src/top/vision/shortcut.h

```diff
@@ -0,0 +1,91 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file shortcut.h
+ */
+#ifndef NNVM_TOP_VISION_SHORTCUT_H_
+#define NNVM_TOP_VISION_SHORTCUT_H_
+
+#include <string>
+#include <vector>
+#include <utility>
+#include <iostream>
+#include <sstream>
+
+namespace nnvm {
+namespace top {
+
+template <typename AttrType, bool (*is_none)(const AttrType &),
+          bool (*assign)(AttrType *, const AttrType &), bool reverse_infer,
+          std::string (*attr_string)(const AttrType &), int n_in = -1,
+          int n_out = -1>
+inline bool
+ShortcutAttr(const nnvm::NodeAttrs &attrs, std::vector<AttrType> *in_attrs,
```

Comment:
if a function contains multiple lines of arguments, always put one line per argument